admission-webhook/Makefile

```diff
@@ -3,7 +3,7 @@ SHELL := /bin/bash
 
 WEBHOOK_ROOT := $(CURDIR)
 
-REGISTRY ?= sigwindowstools
+REGISTRY ?= registry.k8s.io/gmsa-webhook
```

Comment:
Was this change intended?

Reply:
we no longer use dockerhub `sigwindowstools` registry for images, looks like it was an oversight in the make file.